### DISCUSSION TITLE
history or course in folder

### DISCUSSION THREAD

**Discussion by Annahita2004** [BlackIQ/Hello-World#80] [Ideas]
Put useful course in readme
or
history

**Comment by BlackIQ** [comment]
Hello ,

Yes , we will describe lang or put wiki link . but about course , Have no idea .
Who wants to learn Fortran as an example ? And also you know course for Fortran !?
I'll talk about it with Parsa .

Thanks for idea .
Amir .

**Reply by parsampsh** [reply]
Yes. we can write some introductions into README.md about the language. but about the full tutorial, i suggest to list some of great courses on youtube or books and link to them. i think this is better.

**Reply by BlackIQ** [reply]
Of course .
I'll work on that .

**Comment by amireshoon** [comment]
It's better to add some installation or configuration guide for popular languages such as Go.

**Reply by BlackIQ** [reply]
Such a fantastic idea .
Agreed !
